synapse/push/mailer.py

```diff
@@ -890,7 +890,7 @@ def safe_markup(raw_html: str) -> Markup:
                 tags=ALLOWED_TAGS,
                 attributes=ALLOWED_ATTRS,
                 # bleach master has this, but it isn't released yet
-                # protocols=ALLOWED_SCHEMES,
+                protocols=ALLOWED_SCHEMES,
```

Comment:
We still have a minimum version of [`bleach == 1.4.3`](https://github.com/matrix-org/synapse/blob/4ba528d9c31e1af75f3532ac2a52c1b12365f592/pyproject.toml#L193) specified which would also need to be updated. I don't know what the constraint is on why it's set so low at the moment. 

Relevant `bleach` 1.5 changelog that adds the `protocols`option : https://bleach.readthedocs.io/en/latest/changes.html#version-1-5-november-4th-2016

Reply:
No particular constraint AFAICS; see also https://github.com/matrix-org/synapse/pull/14906#discussion_r1086535472